setup/PowerShellSetup.Test/MsiLayoutTests.cs

```diff
@@ -1,274 +0,0 @@
-﻿// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License. See License.txt in the project root for license information.
-
```

Comment:
You will need to remove all the code in build.proj that builds the old MSI (or uses anything that you deleted) in order for this not to break the build.  I would also think you'd want to add a line in build.proj that will build the new MSI.

Reply:
I'm going to leave y'all to adjust the build process and signing. I create

Reply:
Okay sounds good. I’ll submit a change to your branch just to remove all of the old MSI stuff as it won’t build otherwise.